app/assets/stylesheets/alchemy/image_library.scss

```diff
@@ -0,0 +1,137 @@
+.picture-detail-bottom {
+  @include box-sizing(border-box);
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  height: 210px;
+  background-color: $medium-gray;
+  padding-top: 2*$default-padding;
+  @include box-shadow(0 -2px $default-padding -2px $text-color);
+
+  .picture-detail-form {
+    position: relative;
+    left: -75px;
+    @include clearfix;
+    max-width: 1000px;
+    margin: 0 auto;
+  }
+
+  form, .picture-detail-usage {
+    @include box-sizing(border-box);
+    float: left;
+    width: 50%;
+    padding: 0 8px;
+  }
+}
+
+#zoomed_picture_background {
+  @include box-sizing(border-box);
+  width: 100%;
+  height: 100%;
+  padding-top: 33px;
+  padding-bottom: 210px;
+  margin: 0 auto;
+  text-align: center;
+  cursor: pointer;
+
+  &:before {
+    content: '';
+    vertical-align: bottom;
```

Comment:
Properties should be ordered content, display, height, margin-left, vertical-align